acceptance_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2022 Meroxa, Inc.
+// Copyright © 2023 Meroxa, Inc.
```

Comment:
It turned out that this year should be the year of the first publication of the work, i.e. 2022 in this case. The linter, however, is always checking if the year is the current year. In Conduit, we changed the linter check: https://github.com/ConduitIO/conduit/pull/803

Reply:
I changed it, Thanks for the this information